sentry-kotlin-multiplatform/src/commonAppleMain/kotlin/io/sentry/kotlin/multiplatform/extensions/SentryOptionsExtensions.kt

```diff
@@ -28,11 +30,20 @@ internal fun CocoaSentryOptions.applyCocoaBaseOptions(options: SentryOptions) {
     this.enableAutoSessionTracking = options.enableAutoSessionTracking
     this.beforeSend = { event ->
         dropKotlinCrashEvent(event as NSExceptionSentryEvent?) as SentryEvent?
+
+        val cocoaName = BuildKonfig.SENTRY_COCOA_PACKAGE_NAME
+        val cocoaVersion = BuildKonfig.SENTRY_COCOA_VERSION
+        options.sdk.apply {
+            val pkg = Package(cocoaName, cocoaVersion)
+            if (!this.packages.contains(pkg)) {
```

Comment:
I don't think this would work because `contains` is checking a reference of `pkg` here.
You'd need to match the `cocoaName` most likely.